mmhuman3d/core/cameras/camera_parameters.py

```diff
@@ -82,9 +82,9 @@ def __init__(self,
         self.parameters_dict['in_mat'] = in_mat
         for distort_name in __distort_coefficient_names__:
             self.parameters_dict[distort_name] = 0.0
-        self.check_item('H', H)
+        _, H = self.check_item('H', H)
```

Comment:
It is a little weird that `check_item` returns a value, any other suggested name?

Reply:
The first returned element is key, but it is not needed yet.

Reply:
Fixed in 5a4ff7b.